common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala

```diff
@@ -34,7 +35,23 @@ object UpdateMessage {
   implicit val unw = Json.writes[UsageNotice]
   implicit val unr = Json.reads[UsageNotice]
   implicit val writes = Json.writes[UpdateMessage]
-  implicit val reads = Json.reads[UpdateMessage]
+  implicit val reads =
+    (
+      (__ \ "subject").read[String] ~
+        (__ \ "image").readNullable[Image] ~
+        (__ \ "id").readNullable[String] ~
+        (__ \ "usageNotice").readNullable[UsageNotice] ~
+        (__ \ "edits").readNullable[Edits] ~
+        // We seem to get messages from _somewhere which don't have last modified on them.
```

Comment:
Can we log the `subject` of the messages missing `lastModified` to narrow this down and fix?